captainhook_enabled.json

```diff
@@ -0,0 +1,62 @@
+{
+    "commit-msg": {
+        "enabled": true,
+        "actions": [
+            {
+                "action": "\\AppBundle\\GitHooks\\Action\\Regex",
+                "options": {
+                    "regex": "/^[A-Z]+-\\d+:?\\s/",
+                    "error": "Your commit message MUST start with a ticket ID"
+                }
+            }
+        ]
+    },
+    "pre-push": {
+        "enabled": false,
+        "actions": []
+    },
+    "pre-commit": {
+        "enabled": true,
+        "actions": [
+            {
+                "action": "\\AppBundle\\GitHooks\\Action\\CheckForBlacklistedWords",
+                "options": {
+                    "extensions": ["php", "twig"],
+                    "keyword_blacklist": ["var_dump", "dump"],
+                    "whitelisted_files": ["src/AppBundle/GitHooks/Action/CheckForBlacklistedWords.php"]
+                }
+            },
+            {
+                "action": "\\CaptainHook\\App\\Hook\\Composer\\Action\\CheckLockFile"
+            },
+            {
+                "action": "\\AppBundle\\GitHooks\\Action\\CheckLockFileCommitted",
```

Comment:
How about an action to check for `yarn.lock` and (in v2) `symfony.lock`?

Reply:
Isn't `yarn.lock` changed way too often - https://github.com/yarnpkg/yarn/issues/4379?

Also, which changes trigger change in `symfony.lock`?

Reply:
> Isn't `yarn.lock` changed way too often - https://github.com/yarnpkg/yarn/issues/4379?

Still needs to be commited when `package.json` is updated, does it not?

Also, which changes trigger change in `symfony.lock`?

Good question! It is changed when running `composer update` or when updating Flex recipes. It can change without ever touching `symfony.json` or `symfony.lock`, so we can't really check for it. Feel free to disregard.

Reply:
Ok, will add the check for `yarn.lock` if `package.json` has been committed :+1:

Reply:
Added in fa136a3b